src/Kiota.Builder/WorkspaceManagement/WorkspaceConfiguration.cs

```diff
@@ -5,12 +5,16 @@
 
 namespace Kiota.Builder.WorkspaceManagement;
 
-public class WorkspaceConfiguration : ICloneable
+public class WorkspaceConfiguration(string kiotaVersion) : ICloneable
 {
     /// <summary>
     /// The version of the configuration file schema.
     /// </summary>
     public string Version { get; set; } = "1.0.0";
+    /// <summary>
+    /// The version of kiota used by the curent workspace.
+    /// </summary>
+    public string KiotaVersion { get; set; } = kiotaVersion;
```

Comment:
let's remove the property from here. It should be read from the API manifest instead.